mmv1/third_party/terraform/tests/resource_bigquery_table_test.go

```diff
@@ -2095,6 +2104,59 @@ resource "google_bigquery_table" "test" {
 `, datasetID, bucketName, objectName, content, connectionID, projectID, tableID, schema)
 }
 
+func testAccBigQueryTableFromGCSWithSchemaWithConnectionId2(datasetID, tableID, connectionID, projectID, bucketName, objectName, content, schema string) string {
+	return fmt.Sprintf(`
+resource "google_bigquery_dataset" "test" {
+  dataset_id = "%s"
+}
+resource "google_storage_bucket" "test" {
+  name          = "%s"
+  location      = "US"
+  force_destroy = true
+}
+resource "google_storage_bucket_object" "test" {
+  name    = "%s"
+  content = <<EOF
+%s
+EOF
+  bucket = google_storage_bucket.test.name
+}
+resource "google_bigquery_connection" "test" {
+   connection_id = "%s"
+   location = "US"
+   cloud_resource {}
+}
+locals {
+   connection_id_reformatted = google_bigquery_connection.test.name
```

Comment:
I'm curious why this is needed for the test, does it get around some sort of caching behavior? I expected you would be able to run the same config twice to hit the error.

Reply:
@edwardmedia I think you missed this question?

Reply:
We caught up offline, and the different format of `connection_id_reformatted` is needed in the test to check that it doesn't produce a diff. The local var is probably not necessary, but it does match the first config, so it seems fine to keep here.